### PR TITLE
Simplified rewriting manifest

### PR DESCRIPTION
**Pull request by duetosymmetry**
Hiyo!

### CODE REVIEW

File: AndroidManifest.xml [hunk 1]
```diff
@@ -718,7 +718,7 @@
                 <data android:scheme="https" android:host="➹.ws" />
                 <data android:scheme="http"  android:host="➽.ws" />
                 <data android:scheme="https" android:host="➽.ws" />
-                <!-- end-autogenerated -->
```
Comment (nicolassmith) [inline AndroidManifest.xml:721]: it should leave this line unchanged

Comment (duetosymmetry) [inline AndroidManifest.xml:721]: WONTFIX

Comment (nicolassmith) [inline AndroidManifest.xml:721]: U MAD?

**Comment by nicolassmith** [comment]
Can you rewrite this in Haskell? KTHXBAI

**Pull request merged**